libs/SalesforceSDK/src/com/salesforce/androidsdk/security/PasscodeManager.java

```diff
@@ -511,22 +518,23 @@ public void setMinPasscodeLength(int minPasscodeLength) {
              * triggered later from OAuthWebviewHelper.
              */
             if (hasStoredPasscode(SalesforceSDKManager.getInstance().getAppContext())) {
-        		showLockActivity(SalesforceSDKManager.getInstance().getAppContext(),
-        				true);
+                setEnabled(true);
+                showLockActivity(ctx, true);
+                return true;
```

Comment:
@skyler-b Wouldn't this lead to `minPasscodeLength` not being updated? Previously, after the lock activity is shown, control would return here and we would execute `storeMobilePolicy` below. Now, since the code returns here, the min length won't be updated, right?